website/thunderbird/115.0/whatsnew/index.html

```diff
@@ -0,0 +1,289 @@
+      </div>
+    </div>
+  </section>
+
+  <div class="nebula"><img src="/media/img/thunderbird/backgrounds/nebula.png" alt=""></div>
+
+  <section class="whats-new-section">
+    <div class="container accessability">
+      <div class="info">
+        <div class="header">
+          <span class="icon">{{ svg('supernova-handshake') }}</span>
+          <h3>{{_('Expanded Accessability')}}</h3>
+        </div>
+        <p>
+          {{_('Supernova substantially improves Thunderbird’s keyboard navigation and screen reader accessibility across the
+          entire application. We’ve also greatly expanded the ability to navigate Mail content and buttons using the TAB and
+          arrow keys.')}}
+        </p>
+      </div>
+      <div class="image">
+        <div class="toolbar">
+          <button class="button">{{_('Button')}}</button>
+          <div class="button-group roving-group">
+            <button class="button">{{_('Button 1')}}</button>
+            <button class="button">{{_('Button 2')}}</button>
+            <button class="button">{{_('Button 3')}}</button>
+            <button class="button">{{_('Button 4')}}</button>
+          </div>
+        </div>
+        <div class="keyboard-keys">
+          <button class="key">Tab</button>
+          <button class="key">←</button>
+          <button class="key">→</button>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="nebula"><img src="/media/img/thunderbird/backgrounds/nebula.png" alt=""></div>
+
+  <section class="whats-new-section">
+    <div class="container calendar">
+      <div class="info">
+        <div class="header">
+          <span class="icon">{{ svg('supernova-calendar') }}</span>
+          <h3>{{_('Improved Calendar Design')}}</h3>
+        </div>
+        <p>
+          {{_('As part of an ongoing effort to modernize and upgrade Thunderbird’s Calendar, Supernova introduces an improved
+          “mini-month” layout, improvements to the day/week/month grid, a pleasing color palette, and several more minor changes.')}}
+        </p>
+      </div>
+      <div class="image">
+        <img src="/media/img/thunderbird/whatsnew/115/cal.png" alt="">
+      </div>
+    </div>
+  </section>
+
+  <div class="nebula"><img src="/media/img/thunderbird/backgrounds/nebula.png" alt=""></div>
+
+  <section class="whats-new-section">
+    <div class="container info more">
+      <h3>{{_('More to come!')}}</h3>
+      <p>
+        {{_('Supernova is constantly evolving. Throughout the next year, we’ll deliver many improvements to existing Supernova
+        features and introduce brand new ones. Upgrade to version 115 and experience the future of Thunderbird!')}}
+      </p>
+    </div>
+  </section>
+
+  <script>
+    const root = document.documentElement;
+    const slider = document.getElementById("slider");
+    slider.addEventListener("input", (e) => {
+      root.style.setProperty("--slider", `${e.target.value}%`);
+    });
```

Comment:
Indeed I can reproduce the issue reported by @Sancus onw Windows.
The problem is that the `input` field steals the focus on click and scrolling on top of the image makes you interact with the slider.

A simple hack we could try is add a `mouseup` event listener to the slider and move the focus back to the body.
Or maybe on `click` we could store the `document.activeElement` and then restore that focus on `mouseup`.
Try to play with these.